### PR TITLE
Feature mobile

### PR DESCRIPTION
**Pull request by Olivier-Kango**
### Hi there 👋,
In this particular Capstone-Project, I have set up : 

#### The Mobile version 

home-about

- I built these 2 pages:
  - [ ] The _Home page._
  - [ ] The _About page._
  
   They all have links in the mobile menu.

- I implemented the following interactions:
  - [ ] When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
  - [ ] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
  
#### The Desktop Version

Untitled

- In this project :
  - [ ] I have set up the breakpoint to 768px,
  - [ ] I Adapted the 2 pages to desktop,
  - [ ] Included some transitions and animations,

 #### A Dynamic page

Untitled

- In the "Featured Speakers" section, I :
   - [ ] Inserted data dynamically using JavaScript.
   - [ ] Used an array with the data about the speakers.
  
Here is the link where I explained my project in [loom](https://www.loom.com/share/7c1d9d4ca98f456eadf64e3f2b7a1eee).

_Contributions, issues, and feature requests are welcome!_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (ShahierNashaat) [inline README.md]: - [ ] Kindly notice that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## Built With
+
+- Library : Font Awesome
+- Framework : Boostrap
+- Technologies : Html & CSS & JavaScript
+
+
+## Live Demo
+
+[My First Capstone-Project](https://capstone-one.netlify.app/)
```
Comment (ShahierNashaat) [inline README.md]: - [ ] You have done a great job deploying your website using netlify server but it is required to deploy your project using GitHub pages so please change the deployment server.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Olivier-Kango, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Olivier-Kango  🔥 

Good job on implementing the requirements. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**